### DISCUSSION TITLE
[@types/bootstrap] Offcanvas.show() doesn't require a parameter

### DISCUSSION THREAD

**Discussion by cetetesoft** [DefinitelyTyped/DefinitelyTyped#55821] [Issues with a @types package]
TypeScript is complaining about the argument `relatedTarget` of `Offcanvas.show()`. However, this should be marked as optional.

In the other hand, `Modal.show()` should have a similar signature:
`show(relatedTarget?: HTMLElement): void;`

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
To be fixed here: #55827
